### DISCUSSION TITLE
Is this 'CAPITULUM' (U+2E3F)?

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#93] [Q&A]
![Zaborowski1535capitulum](https://user-images.githubusercontent.com/8024702/145757496-0f6152d7-3455-444d-a2c4-9a77bc689107.png)
It occurs quite often in a Latin text printed in Poland.

**Comment by psb1558** [comment]
To me it looks more like U+204C BLACK LEFTWARDS BULLET. But it's not that either. What is this text, and what is the character's function?

**Reply by jsbien** [reply]
The full text is here: https://cyfrowe.mnk.pl/publication/24392Cimelia. For me the function of the character is marking the beginning of a paragraph or a similar unit.
It's quite possible the character occurs also in other items printed by Szarfenberg (a well known printer operating in the 16th century Poland), by I haven't checked.

**Reply by psb1558** [reply]
So an interesting variant of U+00B6 PILCROW SIGN.

The Krakow Museum has a nice collection of Szarfenberg's books online—35 of them. He seems a very good designer--doesn't do the same book over and over, and he's got some beautiful type in his  shop. I saw this symbol in one other book, but I bet it's in more. Is it worth including as a variant of U+00B6? Or maybe of U+2E3F, though I can't think why U+2E3F is in Unicode at all: putting it there would bury it.

<img width="234" alt="image" src="https://user-images.githubusercontent.com/16215767/145802495-aba7f216-fb5f-4c2d-8943-8e743e7863b5.png">

**Reply by jsbien** [reply]
My personal preference is to treat it as a variant of U+2E3F, but I will be happy also with U+00B6.

**Reply by psb1558** [reply]
I'll think about this. I'm not terribly comfortable adding U+2E4E. It [came into Unicode as a wingding or webding](https://www.unicode.org/L2/L2011/11247-n4115-wingdings.pdf), without the usual sourcing. I can't find any authority for the description "ancestor of the pilcrow sign," which seems to me unlikely. This glyph, on the other hand, actually _is_ an ancestor of the pilcrow sign.

**Reply by jsbien** [reply]
I don't remember whether Keith Houston (https://shadycharacters.co.uk/) said anything on this very topic (he wrote the whole chapter on pilcrow), I would have to borrow it again from the library, and at present it is rather cumbersome.

**Reply by psb1558** [reply]
Our copy of Houston is in Special Collections, which I haven't visited since before the pandemic, but there's an article in [Smithsonian Magazine](https://www.smithsonianmag.com/arts-culture/the-origin-of-the-pilcrow-aka-the-strange-paragraph-symbol-8610683/), which I'm about to read.

**Reply by jsbien** [reply]
I just realized that the Polish translation of the book is quite cheap and easily available, I ordered it and expect it soon.

**Reply by psb1558** [reply]
Well, it's short, just a blog post, but it looks as if the earliest (printed) pilcrows were based on an ornamented C for Capitulum, like that used by Szarfenberg. I have no idea how it acquired its P-like stem. I might have to get hold of Houston and find out.

**Reply by jsbien** [reply]
I've just got the book, but it doesn't answer the question. You can try to get the full text of https://www.researchgate.net/publication/333555763_Pause_and_Effect_An_Introduction_to_the_History_of_Punctuation_in_the_West.

**Comment by jsbien** [comment]
I had a closer look at U+204C BLACK LEFTWARDS BULLET and, to my slight surprise, I found that the Stix and Free fonts provides glyphs which are quite acceptable to me (they need to be enlarged, but this is not a serious problem in XeLaTeX). Below please find the glyphs from JunicodeTwoBeta, STIXTwoText-Regular and FreeSerif:
![bullet](https://user-images.githubusercontent.com/8024702/146331360-27b3910f-8207-4632-8c7d-76fd845ffaba.png)
BTW, I will get the Polish translation of Houston in a few hours.

**Comment by marrus-sh** [comment]
I had assumed that U+204C ⁌ BLACK LEFTWARDS BULLET and U+204D ⁍ BLACK RIGHTWARDS BULLET were encoded as straightforward ordinary dingbats, but [the original proposal is actually a bit more specific](http://std.dkuug.dk/jtc1/sc2/wg2/docs/n1748.pdf). Considering U+204D ⁍ BLACK RIGHTWARDS BULLET and U+204B ⁋ REVERSED PILCROW SIGN were encoded at the same time and for the same reason, I don’t think it’s that strange to think of U+204C ⁌ BLACK LEFTWARDS BULLET as a variant of U+00B6 ¶ PILCROW SIGN as well.

**Reply by jsbien** [reply]
I agree. The characters come from ISO 5426-2. From https://en.wikipedia.org/wiki/ISO_5426:

> ISO 5426-2 ("Latin characters used in minor European languages and obsolete typography") is a second part to ISO 5426, published in 1996.[4] It specifies a set of 70 characters, some of which do not exist in Unicode. Michael Everson proposed the missing characters in Unicode 3.0, but some were postponed for further study. Later, new evidence was found, and more was encoded.

**Comment by psb1558** [comment]
(Packaging up a couple answers here.)

This is all very interesting. I can't easily get hold of Pause and Effect, but I think we know enough now. In all probability, U+204C-D are ultimately from early pilcrows like those in Szarfenberg's typefonts, and it would make a good bit of sense for Junicode to add serifs to these.
